src/contracts/mod.rs

```diff
@@ -123,7 +124,7 @@ impl IdentityManager {
     }
 
     #[instrument(level = "debug", skip_all)]
-    pub async fn is_owner(&self) -> anyhow::Result<bool> {
+    pub async fn _is_owner(&self) -> anyhow::Result<bool> {
```

Comment:
Why?

Reply:
They were unused (with and without `oz-provider` feature) which was throwing compilation warnings

Reply:
Let's ✂️ it then. :)